timed/tracking/migrations/0007_migrate_activity_blocks.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+
+def migrate_blocks(apps, schema_editor):
+    Activity = apps.get_model('tracking', 'Activity')
+    for activity in Activity.objects.all():
+        if activity.blocks.count() == 1:
+            block = activity.blocks.all()[0]
+            activity.from_time = block.from_time
+            activity.to_time = block.to_time
+            activity.save()
+        else:
+            for block in activity.blocks.all():
+                Activity(from_time=block.from_time,
```

Comment:
you have to do Activity.objects.create() otherwise nothing is created.

Reply:
Changed it